tools/juliapkg/src/result.jl

```diff
@@ -704,6 +704,7 @@ function execute(stmt::Stmt, params::DBInterface.StatementParams = ())
     if Threads.nthreads() != 1
         # When we have additional worker threads, don't execute using the main thread
         while duckdb_execution_is_finished(stmt.con.handle) == false
+            Base.yield()
```

Comment:
Can we also call `Base.yield` in `execute_tasks` (where we call `GC.safepoint`)?